packages/starters/formstack/mayflower-theme.css

```diff
@@ -419,4 +419,155 @@ button.ma__input-number__minus {
 .fieldset-content:last-child div.horizontal input[type="text"] {
     border: 2px solid rgb(220,220,220);
     width: 100%;
-}
\ No newline at end of file
+}
+
+
+/* Save Incompoete button and Pagination component */
+.page-content .sfFormfs.fsSingleColumn>.fsSaveIncomplete {
+    /* override sf styles */
+    background: none !important;
+    padding: 1em 0;
+    box-shadow: none !important;
+}
+
+div.fsSaveIncomplete a {
+    /* override sf styles */
+    margin: 2em 0;
+    /* button */
+    cursor: pointer;
+    font-family: "Noto Sans VF", "Noto Sans", "Helvetica", "Arial", sans-serif;
+
+    /* ma__button */
+    border: 3px solid;
+    -webkit-box-shadow: 0 0.25rem 0.5rem rgba(1, 1, 1, .25);
+    box-shadow: 0 0.25rem 0.5rem rgba(1, 1, 1, .25);
+    display: -webkit-inline-box;
+    display: -ms-inline-flexbox;
+    display: inline-flex;
+    -webkit-box-align: center;
+    -ms-flex-align: center;
+    align-items: center;
+    -webkit-box-pack: center;
+    -ms-flex-pack: center;
+    justify-content: center;
+    letter-spacing: .1em;
+    padding: 0.4em 1.5em;
+    /* 1em; */
+    text-decoration: none;
+    text-transform: uppercase;
+    -webkit-transition: all .4s ease;
+    -o-transition: all .4s ease;
+    transition: all .4s ease;
+    font-size: 1.25rem;
+    line-height: 1.4;
+    /* ma__button--secondary */
+    font-weight: 550;
+    background-color: #fff;
+    border-color: rgba(20, 85, 143, .5);
+}
+
+.fsForm.fsSingleColumn div.fsSaveIncomplete a {
+    color: #14558f;
+}
+
+.fsForm.fsSingleColumn div.fsSaveIncomplete a:hover,
+.fsForm.fsSingleColumn div.fsSaveIncomplete a:focus {
+    color: #fff;
+    background-color: #14558f;
+}
+
+.fsSaveIncomplete a {
+    display: block;
+    text-decoration: none;
+}
+
+/* pagination component */
+.fsPagination {
```

Comment:
For code sanitary, can we consolidate the .fsPagination and .fsPreviousButton .fsNextButton styles with lines ~184 ~ 226 where those selectors first appear?

Reply:
I feel like some of the overrides might have already existed. I might be wrong, but it will be easier to see if we combine them